mrtt-ui/src/data/socio_indicator.js

```diff
@@ -0,0 +1,198 @@
+export const socioIndicators = [
```

Comment:
this looks like a constant. Should we use UPPER_SNAME_CASE for names? SOCIO_INDICATORS